[2024-10-24T00:00:03.203+0000] {processor.py:161} INFO - Started process (PID=16498) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:00:03.206+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:00:03.271+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:00:03.270+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:00:03.402+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:00:03.383+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:00:03.408+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:00:03.587+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.401 seconds
[2024-10-24T00:00:33.801+0000] {processor.py:161} INFO - Started process (PID=16518) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:00:33.802+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:00:33.804+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:00:33.804+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:00:33.844+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:00:33.834+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:00:33.848+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:00:33.879+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.085 seconds
[2024-10-24T00:01:04.321+0000] {processor.py:161} INFO - Started process (PID=16535) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:01:04.323+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:01:04.325+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:01:04.325+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:01:04.359+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:01:04.353+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:01:04.360+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:01:04.388+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.074 seconds
[2024-10-24T00:01:34.798+0000] {processor.py:161} INFO - Started process (PID=16552) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:01:34.800+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:01:34.802+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:01:34.802+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:01:34.834+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:01:34.826+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:01:34.837+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:01:34.867+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.075 seconds
[2024-10-24T00:02:05.341+0000] {processor.py:161} INFO - Started process (PID=16569) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:02:05.343+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:02:05.345+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:02:05.345+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:02:05.382+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:02:05.374+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:02:05.385+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:02:05.412+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.078 seconds
[2024-10-24T00:02:35.861+0000] {processor.py:161} INFO - Started process (PID=16586) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:02:35.862+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:02:35.867+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:02:35.866+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:02:35.898+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:02:35.892+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:02:35.901+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:02:35.924+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.074 seconds
[2024-10-24T00:03:06.381+0000] {processor.py:161} INFO - Started process (PID=16603) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:03:06.382+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:03:06.385+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:03:06.384+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:03:06.424+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:03:06.415+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:03:06.427+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:03:06.451+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.076 seconds
[2024-10-24T00:03:36.890+0000] {processor.py:161} INFO - Started process (PID=16620) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:03:36.892+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:03:36.895+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:03:36.895+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:03:36.929+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:03:36.924+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:03:36.931+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:03:36.956+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.072 seconds
[2024-10-24T00:04:07.378+0000] {processor.py:161} INFO - Started process (PID=16637) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:04:07.380+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:04:07.383+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:04:07.383+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:04:07.416+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:04:07.410+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:04:07.419+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:04:07.446+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.076 seconds
[2024-10-24T00:04:37.888+0000] {processor.py:161} INFO - Started process (PID=16654) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:04:37.890+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:04:37.892+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:04:37.892+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:04:37.925+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:04:37.918+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:04:37.927+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:04:37.952+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.070 seconds
[2024-10-24T00:05:08.426+0000] {processor.py:161} INFO - Started process (PID=16671) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:05:08.428+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:05:08.431+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:05:08.430+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:05:08.471+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:05:08.462+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:05:08.474+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:05:08.504+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.084 seconds
[2024-10-24T00:05:39.053+0000] {processor.py:161} INFO - Started process (PID=16688) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:05:39.054+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:05:39.057+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:05:39.056+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:05:39.083+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:05:39.077+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:05:39.085+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:05:39.108+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.061 seconds
[2024-10-24T00:06:09.563+0000] {processor.py:161} INFO - Started process (PID=16705) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:06:09.564+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:06:09.566+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:06:09.566+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:06:09.608+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:06:09.598+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:06:09.611+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:06:09.637+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.080 seconds
[2024-10-24T00:06:40.047+0000] {processor.py:161} INFO - Started process (PID=16722) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:06:40.049+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:06:40.051+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:06:40.051+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:06:40.089+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:06:40.078+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:06:40.092+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:06:40.118+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.078 seconds
[2024-10-24T00:07:10.568+0000] {processor.py:161} INFO - Started process (PID=16739) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:07:10.570+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:07:10.574+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:07:10.573+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:07:10.600+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:07:10.596+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:07:10.601+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:07:10.633+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.072 seconds
[2024-10-24T00:07:41.065+0000] {processor.py:161} INFO - Started process (PID=16756) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:07:41.066+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:07:41.069+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:07:41.069+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:07:41.105+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:07:41.099+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:07:41.108+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:07:41.135+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.077 seconds
[2024-10-24T00:08:11.616+0000] {processor.py:161} INFO - Started process (PID=16773) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:08:11.617+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:08:11.619+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:08:11.619+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:08:11.650+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:08:11.644+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:08:11.652+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:08:11.675+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.067 seconds
[2024-10-24T00:08:42.137+0000] {processor.py:161} INFO - Started process (PID=16790) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:08:42.138+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:08:42.141+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:08:42.141+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:08:42.179+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:08:42.170+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:08:42.182+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:08:42.211+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.081 seconds
[2024-10-24T00:09:12.643+0000] {processor.py:161} INFO - Started process (PID=16807) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:09:12.645+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:09:12.648+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:09:12.647+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:09:12.675+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:09:12.668+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:09:12.677+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:09:12.709+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.073 seconds
[2024-10-24T00:09:43.194+0000] {processor.py:161} INFO - Started process (PID=16824) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:09:43.195+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:09:43.198+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:09:43.197+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:09:43.232+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:09:43.227+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:09:43.235+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:09:43.261+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.073 seconds
[2024-10-24T00:10:13.749+0000] {processor.py:161} INFO - Started process (PID=16841) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:10:13.751+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:10:13.754+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:10:13.754+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:10:13.793+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:10:13.785+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:10:13.795+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:10:13.824+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.080 seconds
[2024-10-24T00:10:44.360+0000] {processor.py:161} INFO - Started process (PID=16858) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:10:44.361+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:10:44.364+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:10:44.363+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:10:44.401+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:10:44.393+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:10:44.404+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:10:44.430+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.075 seconds
[2024-10-24T00:11:14.889+0000] {processor.py:161} INFO - Started process (PID=16875) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:11:14.890+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:11:14.893+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:11:14.892+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:11:14.934+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:11:14.925+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:11:14.938+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:11:14.970+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.088 seconds
[2024-10-24T00:11:45.430+0000] {processor.py:161} INFO - Started process (PID=16892) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:11:45.432+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:11:45.434+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:11:45.433+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:11:45.467+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:11:45.460+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:11:45.470+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:11:45.502+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.077 seconds
[2024-10-24T00:12:15.956+0000] {processor.py:161} INFO - Started process (PID=16909) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:12:15.958+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:12:15.961+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:12:15.960+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:12:15.990+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:12:15.985+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:12:15.992+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:12:16.021+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.071 seconds
[2024-10-24T00:12:46.449+0000] {processor.py:161} INFO - Started process (PID=16926) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:12:46.450+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:12:46.453+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:12:46.453+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:12:46.493+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:12:46.483+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:12:46.497+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:12:46.525+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.084 seconds
[2024-10-24T00:13:16.952+0000] {processor.py:161} INFO - Started process (PID=16943) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:13:16.953+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:13:16.955+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:13:16.955+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:13:16.992+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:13:16.984+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:13:16.995+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:13:17.017+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.071 seconds
[2024-10-24T00:13:47.455+0000] {processor.py:161} INFO - Started process (PID=16960) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:13:47.457+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:13:47.459+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:13:47.459+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:13:47.493+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:13:47.485+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:13:47.496+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:13:47.525+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.076 seconds
[2024-10-24T00:14:17.930+0000] {processor.py:161} INFO - Started process (PID=16977) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:14:17.931+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:14:17.934+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:14:17.933+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:14:17.971+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:14:17.962+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:14:17.974+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:14:18.000+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.077 seconds
[2024-10-24T00:14:48.444+0000] {processor.py:161} INFO - Started process (PID=16994) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:14:48.445+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:14:48.448+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:14:48.448+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:14:48.486+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:14:48.478+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:14:48.489+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:14:48.520+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.082 seconds
[2024-10-24T00:15:18.985+0000] {processor.py:161} INFO - Started process (PID=17011) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:15:18.987+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:15:18.990+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:15:18.989+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:15:19.029+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:15:19.021+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:15:19.033+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:15:19.061+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.083 seconds
[2024-10-24T00:15:49.589+0000] {processor.py:161} INFO - Started process (PID=17028) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:15:49.590+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:15:49.593+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:15:49.592+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:15:49.627+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:15:49.619+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:15:49.629+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:15:49.654+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.070 seconds
[2024-10-24T00:16:20.131+0000] {processor.py:161} INFO - Started process (PID=17045) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:16:20.132+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:16:20.135+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:16:20.135+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:16:20.177+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:16:20.166+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:16:20.179+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:16:20.208+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.084 seconds
[2024-10-24T00:16:50.691+0000] {processor.py:161} INFO - Started process (PID=17062) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:16:50.693+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:16:50.696+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:16:50.696+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:16:50.730+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:16:50.723+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:16:50.732+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:16:50.754+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.068 seconds
[2024-10-24T00:17:21.196+0000] {processor.py:161} INFO - Started process (PID=17079) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:17:21.197+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:17:21.200+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:17:21.199+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:17:21.233+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:17:21.226+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:17:21.237+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:17:21.265+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.074 seconds
[2024-10-24T00:17:51.727+0000] {processor.py:161} INFO - Started process (PID=17097) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:17:51.729+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:17:51.732+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:17:51.731+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:17:51.766+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:17:51.759+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:17:51.769+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:17:51.797+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.076 seconds
[2024-10-24T00:18:22.259+0000] {processor.py:161} INFO - Started process (PID=17114) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:18:22.261+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:18:22.263+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:18:22.263+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:18:22.300+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:18:22.292+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:18:22.303+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:18:22.327+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.074 seconds
[2024-10-24T00:18:52.753+0000] {processor.py:161} INFO - Started process (PID=17131) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:18:52.755+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:18:52.757+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:18:52.757+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:18:52.792+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:18:52.785+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:18:52.794+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:18:52.822+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.077 seconds
[2024-10-24T00:19:23.226+0000] {processor.py:161} INFO - Started process (PID=17148) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:19:23.228+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:19:23.230+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:19:23.230+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:19:23.271+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:19:23.261+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:19:23.273+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:19:23.304+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.084 seconds
[2024-10-24T00:19:53.735+0000] {processor.py:161} INFO - Started process (PID=17164) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:19:53.737+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:19:53.740+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:19:53.739+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:19:53.775+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:19:53.767+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:19:53.778+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:19:53.804+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.078 seconds
[2024-10-24T00:20:24.372+0000] {processor.py:161} INFO - Started process (PID=17181) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:20:24.373+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:20:24.376+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:20:24.376+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:20:24.418+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:20:24.409+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:20:24.421+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:20:24.450+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.084 seconds
[2024-10-24T00:20:54.877+0000] {processor.py:161} INFO - Started process (PID=17198) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:20:54.879+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:20:54.881+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:20:54.881+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:20:54.912+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:20:54.907+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:20:54.914+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:20:54.939+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.068 seconds
[2024-10-24T00:21:25.379+0000] {processor.py:161} INFO - Started process (PID=17215) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:21:25.380+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:21:25.383+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:21:25.383+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:21:25.421+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:21:25.414+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:21:25.423+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:21:25.455+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.083 seconds
[2024-10-24T00:21:55.889+0000] {processor.py:161} INFO - Started process (PID=17232) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:21:55.891+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:21:55.894+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:21:55.893+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:21:55.931+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:21:55.924+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:21:55.934+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:21:55.963+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.079 seconds
[2024-10-24T00:22:26.440+0000] {processor.py:161} INFO - Started process (PID=17249) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:22:26.442+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:22:26.445+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:22:26.444+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:22:26.481+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:22:26.472+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:22:26.483+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:22:26.511+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.077 seconds
[2024-10-24T00:22:57.009+0000] {processor.py:161} INFO - Started process (PID=17266) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:22:57.011+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:22:57.013+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:22:57.013+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:22:57.055+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:22:57.046+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:22:57.058+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:22:57.086+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.082 seconds
[2024-10-24T00:23:27.554+0000] {processor.py:161} INFO - Started process (PID=17283) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:23:27.556+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:23:27.560+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:23:27.559+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:23:27.593+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:23:27.587+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:23:27.595+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:23:27.623+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.075 seconds
[2024-10-24T00:23:58.037+0000] {processor.py:161} INFO - Started process (PID=17299) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:23:58.038+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:23:58.041+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:23:58.040+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:23:58.080+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:23:58.072+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:23:58.084+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:23:58.111+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.080 seconds
[2024-10-24T00:24:28.577+0000] {processor.py:161} INFO - Started process (PID=17316) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:24:28.578+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:24:28.580+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:24:28.580+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:24:28.612+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:24:28.604+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:24:28.615+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:24:28.641+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.071 seconds
[2024-10-24T00:24:59.119+0000] {processor.py:161} INFO - Started process (PID=17333) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:24:59.120+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:24:59.123+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:24:59.122+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:24:59.157+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:24:59.149+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:24:59.160+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:24:59.184+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.070 seconds
[2024-10-24T00:25:29.731+0000] {processor.py:161} INFO - Started process (PID=17350) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:25:29.733+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:25:29.735+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:25:29.735+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:25:29.765+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:25:29.759+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:25:29.768+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:25:29.792+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.066 seconds
[2024-10-24T00:26:00.277+0000] {processor.py:161} INFO - Started process (PID=17367) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:26:00.278+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:26:00.281+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:26:00.281+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:26:00.321+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:26:00.312+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:26:00.325+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:26:00.352+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.082 seconds
[2024-10-24T00:26:30.749+0000] {processor.py:161} INFO - Started process (PID=17384) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:26:30.750+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:26:30.753+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:26:30.753+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:26:30.780+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:26:30.776+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:26:30.781+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:26:30.804+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.062 seconds
[2024-10-24T00:27:01.317+0000] {processor.py:161} INFO - Started process (PID=17401) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:27:01.319+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:27:01.321+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:27:01.321+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:27:01.358+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:27:01.351+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:27:01.361+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:27:01.390+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.079 seconds
[2024-10-24T00:27:31.850+0000] {processor.py:161} INFO - Started process (PID=17418) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:27:31.852+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:27:31.854+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:27:31.854+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:27:31.888+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:27:31.882+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:27:31.889+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:27:31.917+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.073 seconds
[2024-10-24T00:28:02.376+0000] {processor.py:161} INFO - Started process (PID=17436) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:28:02.378+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:28:02.381+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:28:02.381+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:28:02.414+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:28:02.409+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:28:02.416+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:28:02.442+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.083 seconds
[2024-10-24T00:28:32.872+0000] {processor.py:161} INFO - Started process (PID=17453) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:28:32.874+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:28:32.876+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:28:32.876+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:28:32.914+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:28:32.905+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:28:32.917+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:28:32.940+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.076 seconds
[2024-10-24T00:29:03.459+0000] {processor.py:161} INFO - Started process (PID=17473) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:29:03.462+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:29:03.465+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:29:03.465+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:29:03.486+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:29:03.481+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:29:03.487+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:29:03.512+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.061 seconds
[2024-10-24T00:29:33.935+0000] {processor.py:161} INFO - Started process (PID=17495) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:29:33.937+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:29:33.939+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:29:33.939+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:29:33.973+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:29:33.963+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:29:33.976+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:29:34.004+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.075 seconds
[2024-10-24T00:30:04.451+0000] {processor.py:161} INFO - Started process (PID=17512) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:30:04.452+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:30:04.455+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:30:04.454+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:30:04.490+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:30:04.482+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:30:04.493+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:30:04.517+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.072 seconds
[2024-10-24T00:30:35.045+0000] {processor.py:161} INFO - Started process (PID=17529) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:30:35.047+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:30:35.049+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:30:35.048+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:30:35.089+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:30:35.082+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:30:35.091+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:30:35.116+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.078 seconds
[2024-10-24T00:31:05.586+0000] {processor.py:161} INFO - Started process (PID=17546) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:31:05.588+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:31:05.590+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:31:05.589+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:31:05.627+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:31:05.618+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:31:05.630+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:31:05.659+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.079 seconds
[2024-10-24T00:31:36.049+0000] {processor.py:161} INFO - Started process (PID=17563) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:31:36.051+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:31:36.054+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:31:36.053+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:31:36.089+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:31:36.082+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:31:36.092+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:31:36.114+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.070 seconds
[2024-10-24T00:32:06.564+0000] {processor.py:161} INFO - Started process (PID=17580) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:32:06.566+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:32:06.568+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:32:06.568+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:32:06.607+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:32:06.598+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:32:06.610+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:32:06.640+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.083 seconds
[2024-10-24T00:32:37.072+0000] {processor.py:161} INFO - Started process (PID=17597) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:32:37.074+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:32:37.077+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:32:37.076+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:32:37.109+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:32:37.102+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:32:37.112+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:32:37.135+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.069 seconds
[2024-10-24T00:33:07.587+0000] {processor.py:161} INFO - Started process (PID=17614) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:33:07.589+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:33:07.592+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:33:07.591+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:33:07.615+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:33:07.611+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:33:07.617+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:33:07.646+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.065 seconds
[2024-10-24T00:33:38.084+0000] {processor.py:161} INFO - Started process (PID=17631) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:33:38.085+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:33:38.087+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:33:38.087+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:33:38.119+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:33:38.111+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:33:38.122+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:33:38.149+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.072 seconds
[2024-10-24T00:34:08.613+0000] {processor.py:161} INFO - Started process (PID=17648) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:34:08.615+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:34:08.617+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:34:08.617+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:34:08.653+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:34:08.645+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:34:08.655+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:34:08.681+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.074 seconds
[2024-10-24T00:34:39.162+0000] {processor.py:161} INFO - Started process (PID=17665) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:34:39.164+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:34:39.166+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:34:39.166+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:34:39.206+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:34:39.200+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:34:39.210+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:34:39.239+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.083 seconds
[2024-10-24T00:35:09.709+0000] {processor.py:161} INFO - Started process (PID=17682) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:35:09.710+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:35:09.713+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:35:09.713+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:35:09.743+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:35:09.736+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:35:09.745+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:35:09.771+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.068 seconds
[2024-10-24T00:35:40.339+0000] {processor.py:161} INFO - Started process (PID=17699) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:35:40.341+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:35:40.343+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:35:40.343+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:35:40.372+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:35:40.364+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:35:40.373+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:35:40.402+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.072 seconds
[2024-10-24T00:36:10.885+0000] {processor.py:161} INFO - Started process (PID=17716) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:36:10.887+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:36:10.889+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:36:10.889+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:36:10.916+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:36:10.912+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:36:10.918+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:36:10.945+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.065 seconds
[2024-10-24T00:36:41.381+0000] {processor.py:161} INFO - Started process (PID=17733) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:36:41.382+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:36:41.385+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:36:41.384+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:36:41.417+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:36:41.410+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:36:41.420+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:36:41.452+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.076 seconds
[2024-10-24T00:37:11.930+0000] {processor.py:161} INFO - Started process (PID=17750) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:37:11.931+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:37:11.934+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:37:11.934+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:37:11.970+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:37:11.963+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:37:11.973+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:37:12.001+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.077 seconds
[2024-10-24T00:37:42.476+0000] {processor.py:161} INFO - Started process (PID=17767) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:37:42.478+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:37:42.480+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:37:42.480+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:37:42.516+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:37:42.510+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:37:42.518+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:37:42.542+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.072 seconds
[2024-10-24T00:38:12.956+0000] {processor.py:161} INFO - Started process (PID=17784) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:38:12.957+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:38:12.960+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:38:12.959+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:38:12.985+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:38:12.979+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:38:12.987+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:38:13.012+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.063 seconds
[2024-10-24T00:38:43.386+0000] {processor.py:161} INFO - Started process (PID=17801) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:38:43.387+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:38:43.391+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:38:43.390+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:38:43.426+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:38:43.418+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:38:43.429+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:38:43.456+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.078 seconds
[2024-10-24T00:39:13.877+0000] {processor.py:161} INFO - Started process (PID=17818) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:39:13.879+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:39:13.882+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:39:13.881+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:39:13.920+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:39:13.914+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:39:13.922+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:39:13.949+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.079 seconds
[2024-10-24T00:39:44.424+0000] {processor.py:161} INFO - Started process (PID=17836) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:39:44.426+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:39:44.428+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:39:44.427+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:39:44.466+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:39:44.459+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:39:44.469+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:39:44.494+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.077 seconds
[2024-10-24T00:40:14.947+0000] {processor.py:161} INFO - Started process (PID=17853) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:40:14.949+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:40:14.951+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:40:14.951+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:40:14.982+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:40:14.978+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:40:14.984+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:40:15.010+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.070 seconds
[2024-10-24T00:40:45.541+0000] {processor.py:161} INFO - Started process (PID=17870) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:40:45.543+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:40:45.546+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:40:45.545+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:40:45.577+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:40:45.570+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:40:45.580+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:40:45.606+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.071 seconds
[2024-10-24T00:41:16.035+0000] {processor.py:161} INFO - Started process (PID=17887) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:41:16.037+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:41:16.040+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:41:16.039+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:41:16.077+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:41:16.069+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:41:16.080+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:41:16.100+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.071 seconds
[2024-10-24T00:41:46.523+0000] {processor.py:161} INFO - Started process (PID=17905) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:41:46.524+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:41:46.526+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:41:46.526+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:41:46.562+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:41:46.552+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:41:46.565+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:41:46.589+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.071 seconds
[2024-10-24T00:42:17.042+0000] {processor.py:161} INFO - Started process (PID=17922) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:42:17.044+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:42:17.047+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:42:17.047+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:42:17.074+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:42:17.069+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:42:17.077+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:42:17.099+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.063 seconds
[2024-10-24T00:42:47.528+0000] {processor.py:161} INFO - Started process (PID=17939) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:42:47.530+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:42:47.532+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:42:47.532+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:42:47.553+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:42:47.548+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:42:47.555+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:42:47.581+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.059 seconds
[2024-10-24T00:43:18.012+0000] {processor.py:161} INFO - Started process (PID=17956) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:43:18.014+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:43:18.016+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:43:18.016+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:43:18.059+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:43:18.048+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:43:18.062+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:43:18.086+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.081 seconds
[2024-10-24T00:43:48.542+0000] {processor.py:161} INFO - Started process (PID=17973) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:43:48.544+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:43:48.546+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:43:48.546+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:43:48.578+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:43:48.570+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:43:48.580+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:43:48.606+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.070 seconds
[2024-10-24T00:44:19.078+0000] {processor.py:161} INFO - Started process (PID=17990) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:44:19.079+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:44:19.082+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:44:19.082+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:44:19.118+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:44:19.111+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:44:19.121+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:44:19.144+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.071 seconds
[2024-10-24T00:44:49.587+0000] {processor.py:161} INFO - Started process (PID=18007) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:44:49.588+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:44:49.590+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:44:49.590+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:44:49.624+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:44:49.618+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:44:49.625+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:44:49.652+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.071 seconds
[2024-10-24T00:45:20.097+0000] {processor.py:161} INFO - Started process (PID=18024) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:45:20.098+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:45:20.101+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:45:20.101+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:45:20.137+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:45:20.129+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:45:20.140+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:45:20.165+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.075 seconds
[2024-10-24T00:45:50.667+0000] {processor.py:161} INFO - Started process (PID=18041) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:45:50.668+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:45:50.672+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:45:50.671+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:45:50.703+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:45:50.696+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:45:50.706+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:45:50.730+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.070 seconds
[2024-10-24T00:46:21.169+0000] {processor.py:161} INFO - Started process (PID=18058) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:46:21.171+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:46:21.174+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:46:21.173+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:46:21.206+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:46:21.200+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:46:21.207+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:46:21.232+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.069 seconds
[2024-10-24T00:46:51.677+0000] {processor.py:161} INFO - Started process (PID=18075) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:46:51.679+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:46:51.682+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:46:51.681+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:46:51.715+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:46:51.709+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:46:51.717+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:46:51.741+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.070 seconds
[2024-10-24T00:47:22.183+0000] {processor.py:161} INFO - Started process (PID=18092) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:47:22.185+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:47:22.188+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:47:22.187+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:47:22.224+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:47:22.216+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:47:22.226+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:47:22.255+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.079 seconds
[2024-10-24T00:47:52.687+0000] {processor.py:161} INFO - Started process (PID=18109) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:47:52.688+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:47:52.691+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:47:52.691+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:47:52.715+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:47:52.710+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:47:52.717+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:47:52.744+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.066 seconds
[2024-10-24T00:48:23.192+0000] {processor.py:161} INFO - Started process (PID=18126) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:48:23.194+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:48:23.196+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:48:23.196+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:48:23.232+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:48:23.226+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:48:23.234+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:48:23.262+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.076 seconds
[2024-10-24T00:48:53.732+0000] {processor.py:161} INFO - Started process (PID=18143) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:48:53.734+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:48:53.736+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:48:53.736+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:48:53.760+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:48:53.756+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:48:53.762+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:48:53.786+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.060 seconds
[2024-10-24T00:49:24.247+0000] {processor.py:161} INFO - Started process (PID=18160) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:49:24.249+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:49:24.252+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:49:24.252+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:49:24.288+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:49:24.282+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:49:24.291+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:49:24.316+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.075 seconds
[2024-10-24T00:49:54.734+0000] {processor.py:161} INFO - Started process (PID=18177) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:49:54.735+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:49:54.737+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:49:54.737+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:49:54.773+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:49:54.765+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:49:54.776+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:49:54.806+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.078 seconds
[2024-10-24T00:50:25.266+0000] {processor.py:161} INFO - Started process (PID=18194) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:50:25.267+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:50:25.272+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:50:25.271+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:50:25.307+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:50:25.300+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:50:25.310+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:50:25.341+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.082 seconds
[2024-10-24T00:50:55.759+0000] {processor.py:161} INFO - Started process (PID=18211) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:50:55.761+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:50:55.763+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:50:55.762+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:50:55.801+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:50:55.794+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:50:55.803+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:50:55.828+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.075 seconds
[2024-10-24T00:51:26.263+0000] {processor.py:161} INFO - Started process (PID=18228) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:51:26.265+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:51:26.267+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:51:26.267+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:51:26.308+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:51:26.298+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:51:26.311+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:51:26.339+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.084 seconds
[2024-10-24T00:51:56.767+0000] {processor.py:161} INFO - Started process (PID=18245) to work on /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:51:56.769+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl-cmp-poc.py for tasks to queue
[2024-10-24T00:51:56.772+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:51:56.772+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:51:56.797+0000] {logging_mixin.py:188} INFO - [2024-10-24T00:51:56.792+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/etl-cmp-poc.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl-cmp-poc.py", line 4, in <module>
    from airflow.providers.oracle.hooks.oracle import OracleHook
ModuleNotFoundError: No module named 'airflow.providers.oracle'
[2024-10-24T00:51:56.799+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl-cmp-poc.py
[2024-10-24T00:51:56.821+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl-cmp-poc.py took 0.060 seconds
